Packs/RecordedFutureASI/README.md

```diff
@@ -0,0 +1,10 @@
+# Pack Documentation
+This pack is used to access Recorded Future ASI data to pull risk rules into XSOAR as incidents.
+
+## Integration
+The integration is used to access the data from the API.
+
+### Available Actions
+* ASI Risk actions
+    * Get the current issues for a project
+    * Get issues added to a project after a certain timestamp
```

Comment:
Suggest also adding the SecurityTrails wording somewhere in case a user searched by the legacy name.

Reply:
great point! done